x-pack/platform/plugins/shared/stack_connectors/server/connector_types/slack_api/service.ts

```diff
@@ -165,46 +190,60 @@ export const createExternalService = (
     }
   };
 
+  const validateChannels = (channels?: string[], allowedList?: string[]) => {
```

Comment:
This function is fundamentally the same as `validateChannelNames` in line 140. Do we need both?